src/pyspiffe/internal/defaults.py

```diff
@@ -0,0 +1,4 @@
+class Defaults(object):
+
+    spiffe_socket_path = "/tmp/agent.sock"
```

Comment:
There is an [open PR](https://github.com/HewlettPackard/py-spiffe/pull/2) from @mchurichi that generates a config object for the library. One thing we mentioned there is the idea of throwing an exception at config phase in case the ENV VAR that holds the SPIFFE endpoint is not set. Following that I think we should not use a default value for the `spiffe_socket_path`.

Do you think that is correct?

Reply:
I think we should follow along what is done with the other libraries to be consistent - it looks like go-spiffe expects either `SPIFFE_ENDPOINT_SOCKET ` or an explicitly coded path with no default as well? If I'm reading that correctly, then I agree and I'll remove the default.

Reply:
Resolved. The environment variable name is currently hardcoded, maybe this should move to a string table at some point. For now, no config object is required and Defaults was removed.